### ISSUE TITLE
iPad 3 - Slow Rendering Issue

### ISSUE THREAD

**Issue by hashaam** [ekurutepe/MPNotificationView#13]
On iPad 3 following line takes around 10-12 seconds to render as it is rendering in complete frame:

**URL**
https://github.com/Moped/MPNotificationView/blob/master/MPNotificationView/MPNotificationView.m#L454

**Code**

``` ios
UIGraphicsBeginImageContextWithOptions(layer.frame.size, NO, scale);
```

To fix this, we need to render only rect.size:

``` ios
UIGraphicsBeginImageContextWithOptions(rect.size, NO, scale);
```

**Comment by ekurutepe** [comment]
good point! I tried that too but it broke other framing related stuff (like orientations, retina/non-retina) so I left it as it is for the time being but please do submit a pull request if you have a working fix.

thanks!
